src/status_im/utils/handlers.cljs

```diff
@@ -29,9 +29,9 @@
     :after
     (fn check-handler
       [context]
-      (let [new-db (get-coeffect context :db)
+      (let [new-db (get-effect context :db)
             v (get-coeffect context :event)]
-        (when-not (s/valid? :status-im.specs/db new-db)
+        (when (and new-db (not (s/valid? :status-im.specs/db new-db)))
```

Comment:
I'd rather drop the `and` unless there is a good reason: if `db` gets nilified we won't see it right away.